.gitignore

```diff
@@ -0,0 +1,20 @@
+# Ignore Visual Studio Code settings
```

Comment:
Why is the `.gitignore` file part of this [PR](https://github.com/Muhammadali2025222/C-Practice/pull/1) too: